src/components/record/record-footer.tsx

```diff
@@ -0,0 +1,18 @@
+import '@isrd-isi-edu/chaise/src/assets/scss/_record.scss';
```

Comment:
We already have a `footer` component. Please make sure you're looking at recordset implementation and use the same components if they already exits.